### PR TITLE
Fix content type errors by not fixing it

### PR DESCRIPTION
**Pull request by alexduf**
I'm seeing a content type error for w10 notification.

We have dropped the windows 10 app support for long enough, this will end support for their push notifications.

To be merged on Monday

### CODE REVIEW

**Comment by NathanielBennett** [comment]
:+1: Mostly looks good. One Q re one of the test changes . Answer and I'll approve

File: registration/conf/routes [hunk 1]
```diff
@@ -1,4 +1,3 @@
-PUT         /registrations/:lastKnownDeviceId    registration.controllers.Main.register(lastKnownDeviceId)
```
[the remark this answers is not in the document]
Comment (NathanielBennett) [reply registration/conf/routes]: :+1:

File: registration/test/registration/controllers/MainControllerSpec.scala [hunk 2]
```diff
@@ -81,87 +68,6 @@ class MainControllerSpec extends PlaySpecification with JsonMatchers with Mockit
...
-
-      val Some(result) = route(app, FakeRequest(GET, "/registrations?udid=83B148C0-8951-11E5-865A-222E69A460B9"))
-      status(result) must equalTo(OK)
-      contentAsString(result) must /#(0) /("userId" -> "83b148c0-8951-11e5-865a-222e69a460b9")
-    }
-
-    "return registrations for device token with platform" in new RegistrationsContext {
-      val Some(register) = route(app, FakeRequest(PUT, "/registrations/someId").withJsonBody(Json.parse(registrationJson)))
-      status(register) must equalTo(OK)
-
```
[the remark this answers is not in the document]
Comment (alexduf) [reply registration/test/registration/controllers/MainControllerSpec.scala]: I kept the one that was relevant now

**Comment by alexduf** [comment]
@markrichardsg Yes in that case it's to parse a response from Azure, so technically we can have that as part of the response

**Pull request merged**